sympy/core/symbol.py

```diff
@@ -326,7 +347,7 @@ def __xnew_cached_(cls, name, **assumptions):  # symbols are always cached
         return Symbol.__xnew__(cls, name, **assumptions)
 
     def __getnewargs_ex__(self):
-        return ((self.name,), self.assumptions0)
+        return ((self.name,), self._assumptions_orig)
```

Comment:
This is not strictly needed. It came up because the pickling test were comparing `_assumptions_orig` before and after pickling. This preserves that attribute but also is more correct for the meaning of `__getnewargs_ex__` since these are the arguments that created this object.

This also makes pickles smaller:
```python
In [3]: len(pickle.dumps(Symbol('x', positive=True)))
Out[3]: 66
```
The equivalent with master is:
```python
In [2]: len(pickle.dumps(Symbol('x', positive=True)))
Out[2]: 337
```
You can see why in the pickled output:

> \x80\x04\x95F\x01\x00\x00\x00\x00\x00\x00\x8c\x11sympy.core.symbol\x94\x8c\x06Symbol\x94\x93\x94\x8c\x01x\x94\x85\x94}\x94(\x8c\x08positive\x94\x88\x8c\x11extended_positive\x94\x88\x8c\x04real\x94\x88\x8c\x0bcommutative\x94\x88\x8c\x08infinite\x94\x89\x8c\timaginary\x94\x89\x8c\x07complex\x94\x88\x8c\x14extended_nonnegative\x94\x88\x8c\x07nonzero\x94\x88\x8c\rextended_real\x94\x88\x8c\x08negative\x94\x89\x8c\x0bnonpositive\x94\x89\x8c\x0bnonnegative\x94\x88\x8c\x10extended_nonzero\x94\x88\x8c\x04zero\x94\x89\x8c\x14extended_nonpositive\x94\x89\x8c\x06finite\x94\x88\x8c\x11extended_negative\x94\x89\x8c\thermitian\x94\x88u\x92\x94.